internal/template/time_series_chart_component.go

```diff
@@ -286,6 +315,25 @@ func TimeSeriesChart(series TimeSeries) templ.Component {
 
 	}
 
+	cutoffXValue := cutoffIndex - 1
+
+	cutoffLine := chart.ContinuousSeries{
+		XValues: []float64{float64(cutoffXValue) + 0.75, float64(cutoffXValue) + 0.75},
+		YValues: []float64{0, YAxisValues[len(YAxisValues)-1]},
+		Style: chart.Style{
+			StrokeWidth: 2.0,
+			StrokeColor: drawing.ColorFromHex("cd5c5c"),
+		},
+	}
+	graph.Series = append(graph.Series, cutoffLine)
+
+	cutoffLabel := label{
+		x:    int((float64(cutoffXValue) + 0.75) * (620 / float64(len(series.Weeks)))),
```

Comment:
Create a ticket for fixing magic numbers in here.

Reply:
https://app.plane.so/crocoder/projects/51e10b91-414f-43e0-95f3-3608946536cc/issues/e454830e-ab91-4a92-b3b2-098f778f5252